Cloud Application Manager/Deploying Anywhere/using-private-cloud-on-vmware-cloud-foundation.md

```diff
@@ -0,0 +1,115 @@
+| vApp Template | Select a vApp template from the catalog. VMs imported as OVFs then uploaded as templates are also listed. |
+| Customization Password | Optionally, provide a custom VM administrator password. If you leave this empty, the default administrator password from the template applies.|
+| Instances | Specify the number of instances to spawn from the same template. |
+
+#### Placement
+
+| Option | Description |
+|--------|-------------|
+| Virtual Data Center |	Select the virtual data center from vCloud Foundation. |
+| Network |	Select the network for the virtual data center. |
+| IP Allocation | Assign IP addresses to the VM from the network attached to the virtual data center.<li>**DHCP.** Select this option to get an IP address dynamically where the network gateway acts as a DHCP router.</li><li>**Static IP Pool.** Select this option to get an IP address from a range of IP addresses configured for the network.</li><li>**Static Manual.** If you select this option, Cloud Application Manager finds the next available IP address from the virtual data center network and requests vCloud Foundation to assign it to the VM.</li> |
+| Storage Profile | Choose a [storage option](https://docs.vmware.com/en/VMware-Cloud-Foundation/3.5/com.vmware.vcf.ovdeploy.doc_35/GUID-E6C1B2F8-F6CB-4017-A4D1-53103FF9AB84.html) for vCloud Foundation.<li>**Standard.** Persistent block storage for occasional or event driven data access.</li><li>**SSD-Accelerated.** Higher performance block storage.</li> |
+
+#### Compute
+
+| Option | Description |
+|--------|-------------|
+| Virtual CPUs | Specify the number of CPUs. |
+| Memory | Specify the virtual RAM in GB. |
+
+See the [VMware Cloud Foundation](https://docs.vmware.com/en/VMware-Cloud-Foundation/3.5/com.vmware.vcf.ovdeploy.doc_35/GUID-F2DCF1B2-4EF6-444E-80BA-8F529A6D0725.html) documentation.
+
+#### Disks
+
+| Option | Description |
+|--------|-------------|
+| Disk | Specify the storage capacity in GB. Adding disk storage capacity is supported. |
+
+See the [VMware Cloud Foundation](https://docs.vmware.com/en/VMware-Cloud-Foundation/3.5/com.vmware.vcf.ovdeploy.doc_35/GUID-F2DCF1B2-4EF6-444E-80BA-8F529A6D0725.html) documentation.
+
+### Naming vCloud resources
+
+When creating an instance, an automatic name is generated and used for VApps, VMs and Hostname, all with the same generated name. This way you can identify unregistered instances.
+
+This name generated has maximum length of 15 characters (max allowed in windows hostnames), and it's composed by this values:
+
+```
+InstanceName-ServiceId-InstanceNumber
+```
+
+In the Policy Box it's possible specify you want more that one instance. An example is “jdedepl-i8oq0-1”.  
+
+
+Once created this name, **modifying the vApp name is not supported**, because it could affect to the instance operations. However, it's possible to change manually this name on VMs and Hostnames.  
```

Comment:
```suggestion
Once deployed, **modifying the vApp name is not supported**, because it could affect the instance operations through Cloud Application Manager. However, it's possible to change this name manually on VMs and Hostnames.  
```